### PR TITLE
Added description of clear sessions call.

### PR DESCRIPTION
**Pull request by nikolaystoilov-okta**
Added description on the API call to clear user sessions. 
Updated the users postman collection with the new call.

Resolves: OKTA-99060

Reviewers:
@srinivasanagandla-okta @tthompson-okta @mystiberry-okta

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@nikolaystoilov-okta Just waiting for your other reviewers to approve or comment before merging.

File: _source/_docs/api/resources/users.md [hunk 1]
```diff
@@ -2309,7 +2309,8 @@ Content-Type: application/json
 
 <span class="api-uri-template api-uri-delete"><span class="api-label">DELETE</span> /api/v1/users/*:uid*/sessions
 
-Removes all active user sessions. This will force the user to authenticate on the next opeartion.
+Removes all active identify provider sessions. This will force the user to authenticate on the next opeartion.
```
Comment (johngronberg-okta) [inline _source/_docs/api/resources/users.md]: Shouldn't this be "identity providers", not "identify providers" ?

Comment (mystiberry-okta) [inline _source/_docs/api/resources/users.md]: Also, "operation" not "opeartion"

**Comment by mystiberry-okta** [comment]
@nikolaystoilov-okta Should I merge this for tomorrow's site refresh? Not sure if you're waiting for additional reviews or not.

**Pull request merged**